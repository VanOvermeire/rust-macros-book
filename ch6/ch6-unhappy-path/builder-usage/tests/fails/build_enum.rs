use builder_macro::Builder;

#[derive(Builder)]
pub enum ExampleEnum {}

fn main() {}