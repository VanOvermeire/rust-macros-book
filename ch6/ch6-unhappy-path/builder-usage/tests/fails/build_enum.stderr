error: proc-macro derive panicked
 --> tests/fails/build_enum.rs:3:10
  |
3 | #[derive(Builder)]
  |          ^^^^^^^
  |
  = help: message: not implemented: only implemented for structs
